loops.js

```diff
@@ -29,12 +38,27 @@
 // 3 is oneven
 // .... etc.
 // 15 is oneven
+//loop voor 0-15
+for (let i = 0; i < 16; i++) {
```

Comment:
Ziet er goed uit alleen hou ik het liefst zo kort mogelijk zoals het volgende:
```suggestion
for (let i = 0; i < 16; i++) {
  if (i % 2 === 0) {
    console.log(i + ' is even');
  }
  if (i % 2 !== 0) {
    console.log(i + ' is oneven');
  }
}
```

Reply:
Je zegt hier zolang de rest 0 is zijn de getallen even, is de rest niet 0 is het oneven